package/boot/uboot-envtools/files/ramips

```diff
@@ -36,6 +36,7 @@ zbtlink,zbt-wg2626)
 linksys,ea7500-v2|\
 xiaomi,mir3p|\
 xiaomi,mir3g|\
+xiaomi,mi-router-ac2100|\
```

Comment:
Sort, please :)

Reply:
Thanks. I missed that. Will be pushed in next commit.